### PR TITLE
first commit

### PR DESCRIPTION
**Pull request by VitaliiSorokin**

### CODE REVIEW

File: ConsoleApp1/Helpers.cs [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace ConsoleApp1
+{
+    class Helpers
+    {
+        //public static void DeleteWrongCars()
```
Comment (P3ar1baron) [inline ConsoleApp1/Helpers.cs]: For what do you need commented code ?

File: ConsoleApp1/Helpers.cs [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+        //        Console.WriteLine(car.Attributes["new_productiondate"].ToString());
+        //    }
+        //    Console.WriteLine(i);
+        //    Console.Read();
+        //} // TODO: to delete?
+
+        public static DateTime GenerateDateInRange(DateTime startDate, DateTime endDate, Random rnd)
+        {
+            TimeSpan timeSpan = endDate - startDate;
+            TimeSpan newSpan = new TimeSpan(0, rnd.Next(0, (int)timeSpan.TotalMinutes), 0);
```
Comment (P3ar1baron) [inline ConsoleApp1/Helpers.cs]: what does newSpan variable name mean ?

File: ConsoleApp1/Helpers.cs [hunk 3]
```diff
@@ -0,0 +1,91 @@
...
+        //        //crmSvc.Delete("new_car", new Guid(car.Attributes["new_carid"].ToString()));
+        //        Console.WriteLine(car.Attributes["new_vin"].ToString());
+        //        Console.WriteLine(car.Attributes["new_purchase"].ToString());
+        //        Console.WriteLine(car.Attributes["new_productiondate"].ToString());
+        //    }
+        //    Console.WriteLine(i);
+        //    Console.Read();
+        //} // TODO: to delete?
+
+        public static DateTime GenerateDateInRange(DateTime startDate, DateTime endDate, Random rnd)
```
Comment (P3ar1baron) [inline ConsoleApp1/Helpers.cs]: better to use random instead of rnd

File: ConsoleApp1/Helpers.cs [hunk 4]
```diff
@@ -0,0 +1,91 @@
...
+
+        public static Guid GetCarClassId(List<new_carclass> classes, Random rnd)
+        {
+            int index = rnd.Next(classes.Count);
+            return classes[index].Id;
+        }
+
+        public static Guid GetCarIdByClassId(Guid classId, List<new_car> cars, Random rnd)
+        {
+            List<new_car> filteredCars = (List<new_car>)cars.Where(c => c.new_CarClass.Id.Equals(classId)).ToList();
```
Comment (P3ar1baron) [inline ConsoleApp1/Helpers.cs]: better to move equals and tolist to next line

File: ConsoleApp1/Helpers.cs [hunk 5]
```diff
@@ -0,0 +1,91 @@
...
+            int index = rnd.Next(filteredCars.Count);
+            return filteredCars[index].Id;
+        }
+
+        public static Guid GetCustomerId(List<Contact> contacts, Random rnd)
```
Comment (P3ar1baron) [inline ConsoleApp1/Helpers.cs:40]: same for rnd here, please change to random

File: ConsoleApp1/Helpers.cs [hunk 6]
```diff
@@ -0,0 +1,91 @@
...
+        {
+            int index = rnd.Next(contacts.Count);
+            return contacts[index].Id;
+        }
+
+        public static int GetLocationValue(Random rnd)
+        {
+            List<int> locValues = new List<int>() {
```
Comment (P3ar1baron) [inline ConsoleApp1/Helpers.cs]: please rename to locationValues

File: ConsoleApp1/Program.cs [hunk 7]
```diff
@@ -0,0 +1,258 @@
...
+                        DateTime reservedPickup =
+                            Helpers.GenerateDateInRange(
+                                new DateTime(Constants.RentDuration.YearFrom, Constants.RentDuration.MonthFrom, Constants.RentDuration.DayFrom),
+                                new DateTime(Constants.RentDuration.YearTo, Constants.RentDuration.MonthTo, Constants.RentDuration.DayTo),
+                                random);
+                        DateTime reservedReturn = reservedPickup.Add(Helpers.GenerateRentDuraton(random));
+                        Guid classId = Helpers.GetCarClassId(carClasses, random);
+                        Guid carId = Helpers.GetCarIdByClassId(classId, cars, random);
+                        Guid customerId = Helpers.GetCustomerId(contacts, random);
+                        int pickupLocValue = Helpers.GetLocationValue(random);
```
Comment (P3ar1baron) [inline ConsoleApp1/Program.cs]: please rename to pickupLocationValue and same for line below

File: ConsoleApp1/Program.cs [hunk 8]
```diff
@@ -0,0 +1,258 @@
...
+                            Id = rentId,
+                            statecode = new_rentState.Inactive,
+                            statuscode = new OptionSetValue(Constants.RentStatus.Canceled),
+                        };
+
+                        UpdateRequest updateRentRequest = new UpdateRequest { Target = newRent };
+                        multipleRequest.Requests.Add(updateRentRequest);
+                    }
+
+                    var mltplResponse = (ExecuteMultipleResponse)crmSvc.Execute(multipleRequest);
```
Comment (P3ar1baron) [inline ConsoleApp1/Program.cs]: don't understand the name of the variable here

File: ConsoleApp1/Helpers.cs (hunk 2 above)
Comment (VitaliiSorokin) [reply ConsoleApp1/Helpers.cs]: Just a TimeSpan with random duration in between the given dates.

**Pull request merged**